src/Modules/VRDistortionModule/VRCardboardDeviceProfile.cpp

```diff
@@ -0,0 +1,249 @@
+// Copyright eeGeo Ltd (2012-2016), All Rights Reserved
+
+#include "VRCardboardDeviceProfile.h"
+#include "Logger.h"
+
+namespace Eegeo
+{
+    namespace VR
+    {
+        namespace Distortion
+        {
+            
+            VRCardboardDeviceProfile::VRCardboardDeviceProfile(){
```

Comment:
Probably better to put the default parameters in a factory method and pass them into the constructor as named parameters (through existing VR* structs or otherwise).